src/Agent.Listener/Program.cs

```diff
@@ -88,6 +89,20 @@ private async static Task<int> MainAsync(IHostContext context, string[] args)
                         terminal.WriteError(StringUtil.Loc("MinimumNetFramework"));
                         // warn only, like configurationmanager.cs does. this enables windows edition with just .netcore to work
                     }
+
+                    // Upgrade process priority to avoid Listener starvation
+                    using (Process p = Process.GetCurrentProcess())
+                    {
+                        try
+                        {
+                            p.PriorityClass = ProcessPriorityClass.High;
```

Comment:
Would recommend “AboveNornal” instead of “High”

Reply:
Great catch, we should definitely do that fixing.